PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/projects/MavenProjectGenerator.java

```diff
@@ -114,6 +114,9 @@ private void createPom(String root) throws Exception {
             case SPARK_2_2_0:
                 file = StreamUtil.getResourceFile("/hdinsight/templates/pom/spark_2_2_0_pom.xml");
                 break;
+            case SPARK_2_3_0:
+                file = StreamUtil.getResourceFile("/hdinsight/templates/pom/spark_2_3_0_pom.xml");
+                break;

```

Comment:
I suppose there is a sbt project generator also needed to change

Reply:
Actually sbt project is auto-generated, you can refer to [SbtProjectGenerator.java](https://github.com/Microsoft/azure-tools-for-java/blob/develop/PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/projects/SbtProjectGenerator.java)